orquesta/statuses.py

```diff
@@ -18,6 +18,7 @@
 PAUSING = 'pausing'
 PAUSED = 'paused'
 RESUMING = 'resuming'
+RERUNNING = 'rerunning'
```

Comment:
Remove if these statuses are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.